pyfftw/interfaces/_utils.py

```diff
@@ -97,24 +101,25 @@ def _Xfftn(a, s, axes, overwrite_input, planner_effort,
         # take the transform of the input array! (in general, we have
         # to assume that the input array will be destroyed during 
         # planning).
-        a_copy = a.copy()
+        if overwrite_input:    # Note: reload_after_transform is not True...
+            args = (a, s, axes) + args
+            a_copy = a.copy()  # ... hence this is not an additional copy
+        else:
+            args = (numpy.empty_like(a), s, axes) + args
```

Comment:
I know that the strides are different, but the `fftn` call still works even though the plan is created on an array with different strides than it is executed on.  Can you find a case that fails?  (I would really be interested in this because I have always worried about planning on one array and executing the transform on another, but have never had a problem.  Perhaps it is just a performance issue: the optimal plan is not generated?  That might be pretty tricky to check unless there is a way to inspect the plans...